src/expressions/loop-expr.md

```diff
@@ -308,6 +308,9 @@ Example:
 r[expr.loop.break.value]
 A `break` expression is only permitted in the body of a loop, and has one of the forms `break`, `break 'label` or ([see below](#break-and-loop-values)) `break EXPR` or `break 'label EXPR`.
 
+r[expr.loop.break.type]
+A `break` expression itself has a type of [`!`](../types/never.md).
```

Comment:
Probably I'd locate this rule closer to the existing text:

> When `break` is encountered, execution of the associated loop body is immediately terminated...

I.e., that other rule is already describing divergence.

Have a look at this as well, if you would, for the other similar rules for other expressions.